solidity/dashboard/src/connectors/explorerModeSubprovider.js

```diff
@@ -0,0 +1,22 @@
+import { Subprovider } from "@0x/subproviders";
+
+
+class ExplorerModeSubprovider extends Subprovider {
```

Comment:
I would consider inheriting from `EventEmitter` then we can get rid of `eventEmitter` as a field in the class. We would have to do `this.explorerSubproivder.eventEmitter.on()` to handle subprovider events which is a little weird and could be misleading. Also not sure if we really need `Subprovider` from `@0x/subproviders`.

Reply:
Yeah i used it like that mainly because I wanted the `ExplorerModeSubprovider` to inherit from `@0x/subprovider` which seems like a good idea, because it  make sure that it forces us to have the `handleRequest` method defined 🤔

Reply:
Maybe we can use `mixins` concept? Please see https://javascript.info/mixins.